docs/modules/faqs.html

```diff
@@ -100,17 +100,40 @@ <h3>I have a question not addressed in this FAQ. What is my next step?</h3>
-					<a href="#how-can-i-learn-about-interruptions-to-the-amazon-chime-service" id="how-can-i-learn-about-interruptions-to-the-amazon-chime-service" style="color: inherit; text-decoration: none;">
-						<h3>How can I learn about interruptions to the Amazon Chime service?</h3>
+					<a href="#i-am-unable-to-select-audio-output-device-in-some-browsers-is-this-a-known-issue" id="i-am-unable-to-select-audio-output-device-in-some-browsers-is-this-a-known-issue" style="color: inherit; text-decoration: none;">
+						<h3>I am unable to select audio output device in some browsers, is this a known issue?</h3>
 					</a>
-					<p>You can be notified about Amazon Chime service interruptions at the <a href="https://status.aws.amazon.com/">https://status.aws.amazon.com/</a> website. In addition you can set up <a href="https://aws.amazon.com/cloudwatch/">Amazon CloudWatch</a> events via the <a href="https://aws.amazon.com/premiumsupport/technology/personal-health-dashboard/">AWS Personal Health Dashboard</a> for the Amazon Chime service. You can find the Amazon Chime SLA on this <a href="https://aws.amazon.com/chime/sla/">webpage</a>.</p>
+					<p><a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1152401">Firefox</a> and <a href="https://bugs.webkit.org/show_bug.cgi?id=179415">Safari</a> have known issues disallowing them from listing audio output devices on these browsers. While clients can continue the meeting using the default device, they will not be able to select devices in meetings.</p>
+					<a href="#my-video-disappears-in-safari-browsers-is-this-a-known-issue" id="my-video-disappears-in-safari-browsers-is-this-a-known-issue" style="color: inherit; text-decoration: none;">
+						<h3>My video disappears in Safari browsers, is this a known issue?</h3>
+					</a>
+					<p>macOS and iOS Safari browsers have limitations when you use a camera in other apps and tabs.
+						For example, if you enable a video in two macOS Safari tabs, one video will go black.
+					Make sure to close all other apps and tabs that are using the camera.</p>
+					<a href="#i-cannot-join-meeting-in-firefox-with-no-audio-and-video-permission-due-to-no-ice-candidates-were-gathered-error-is-this-a-known-issue" id="i-cannot-join-meeting-in-firefox-with-no-audio-and-video-permission-due-to-no-ice-candidates-were-gathered-error-is-this-a-known-issue" style="color: inherit; text-decoration: none;">
+						<h3>I cannot join meeting in Firefox with no audio and video permission due to <code>no ice candidates were gathered</code> error, is this a known issue?</h3>
+					</a>
+					<p>Please check your VPN setting. This can happen in some VPN setting such as split tunneling.
+						In Firefox, if you have given access to cam/mic and have not set default_address_only then ICE will gather all interface addresses.
```

Comment:
expand cam/mic

Reply:
THis needs a little rework with more details - i have taken a step but am still not clear(could you clarify how default_address_only is or what default interface is set - am not clear so making assumptions):

In Firefox, if access to camera or microphone has been granted, and the administrator has not set default_address_only then ICE will gather all interface addresses. Without the right permissions,  e.g. data channel only, the ICE stack behaves will limit ICE candidates to a default interface only. This will cause ice gathering step to fail if the SDK application is not loaded in the same network interface that will be used for the SDK.This is a privacy mechanism in Firefox to stop sites from finding out your IP addresses via WebRTC (Is there a link???).